cmake/modules/FindPackageHandleStandardArgs.cmake

```diff
@@ -0,0 +1,386 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
please don't include this module if the existing one in cmake3 is good enough.

Reply:
@tchaikov 
The `FindPackageHandleStandardArgs.cmake ` lloks like to be enough
I added this one since I got missing error in the previous run here, but I guess that could be due to the `${CMAKE_CURRENT_LIST_DIR}` prefix
So I'll try again.

Reply:
@wjwithagen i'd suggest reconsider https://github.com/ceph/ceph/pull/31317#discussion_r341531353 .